webapp/codesearch/settings.py

```diff
@@ -93,7 +102,7 @@
 
 LOGGING = {
     'version': 1,
-    # 'disable_existing_loggers': False,
+    'disable_existing_loggers': False,
```

Comment:
?

Reply:
Some `INFO` level logs weren't showing after the bump. And that changed after this was uncommented. My suspicion is the default behavior for this attribute changed during one of the subsequent releases. @stantonk

Reply:
coolio